src/Microsoft.ML.Core/Properties/AssemblyInfo.cs

```diff
@@ -6,11 +6,16 @@
 using Microsoft.ML;
 
 [assembly: InternalsVisibleTo(assemblyName: "Microsoft.ML.TestFramework" + PublicKey.TestValue)]
+[assembly: InternalsVisibleTo(assemblyName: "Microsoft.ML.Tests" + PublicKey.TestValue)]
+[assembly: InternalsVisibleTo(assemblyName: "Microsoft.ML.Core.Tests" + PublicKey.TestValue)]
 [assembly: InternalsVisibleTo(assemblyName: "Microsoft.ML.Predictor.Tests" + PublicKey.TestValue)]
+[assembly: InternalsVisibleTo(assemblyName: "Microsoft.ML.InferenceTesting" + PublicKey.TestValue)]
+[assembly: InternalsVisibleTo(assemblyName: "Microsoft.ML.StaticPipelineTesting" + PublicKey.TestValue)]
```

Comment:
The one thing I am concerned about with this plan is that our tests will be able to reference the internal types in our product. This means that we won't be able to ensure that we are exposing the necessary public API for our customer scenarios.

Is it possible to only use IVT between our product assemblies, but not between our tests assemblies and the product?